### DISCUSSION TITLE
two questions

### DISCUSSION THREAD

**Discussion by SylvainArd** [ImageMagick/ImageMagick#5055] [Help]
Hello I have two questions please : 
1 : in silent command line install of imagemagick how to check the checkbox "add convert and other utilities" ?
2 : what is the imagemagick command to convert a TIFF file into BMP please ?
Thank you
Best regards

**Comment by fmw42** [comment]
1) What is your platform?  Is that Windows?  If so, I am not an expert, but when you dowloaded the binary installation, and started it, there should have been a checkbox to install the older legacy components for IM 6 in addition to the regular install for IM 7. You do not have to install the legacy components. They are optional if you need backward compatibility. The newer IM 7 is more functional.

2) In command line mode for IM 7,

`magick image.tiff image.bmp`

in legacy mode for IM 6,

`convert image.tiff image.bmp`

A Windows expert will be able to discuss this in more detail or correct my comments.

**Comment by SylvainArd** [comment]
thank you but I found it in the documentation :
ImageMagick-x.y.z-x64-x.exe /VERYSILENT /NORESTART
/MERGETASKS=legacy_support
indeed my question was for silent install
Sylvain Ard
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers


Le mar. 19 avr. 2022 à 16:41, Fred Weinhaus ***@***.***> a
écrit :
